plenum/server/replica.py

```diff
@@ -204,7 +210,7 @@ def __init__(self, node: 'plenum.server.node.Node', instId: int,
         # viewNo and value a map of pre-prepare sequence number to commit
         self.stashed_out_of_order_commits = {}  # type: Dict[int,Dict[int,Commit]]
 
-        self.checkpoints = SortedDict(lambda k: k[0])
+        self.checkpoints = SortedDict(lambda k: k[1])
```

Comment:
`k` is three phase key, `(view_no, pp_seq_no)`. Sorting is on the basis of both of these, for more have a look at `test_3PC_key_comaparison`